README.md

```diff
@@ -24,8 +24,9 @@ Currently supported providers are: [GitHub](#github), [Bitbucket Server](#bitbuc
   - [Delete Webhook](#delete-webhook)
   - [Set Commit Status](#set-commit-status)
   - [Create Pull Request](#create-pull-request)
-  - [Latest Commit Hash](#get-latest-commit-hash)
+  - [Latest Commit](#get-latest-commit)
```

Comment:
Let's put these two together, as there is a logical connection between them. Let's do it also on their description.
```suggestion
  - [Latest Commit](#get-latest-commit)
  - [Get Commit By SHA](#get-commit-by-sha)
```